### PR TITLE
Release Log Updates Until Subsequent Release

### PR DESCRIPTION
**Pull request by nairashwin952013**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,9 +2,39 @@
 All notable changes to this project will be documented in this file.
 This project aims to adhere to [Semantic Versioning](http://semver.org/).
 
-## [3.3.2] -
+## [3.4.0] -
+### Added
+ - [Updated Prune-Depth Documentation. Add Unit Tests for more limit values](https://github.com/joyent/java-manta/pull/484)
 ### Fixed
- - [Content-Type is not set to application/octet-stream when doing CSE](https://github.com/joyent/java-manta/issues/486) 
+ - [Content-Type is not set to application/octet-stream when doing CSE](https://github.com/joyent/java-manta/issues/486)
```
Comment (dekobon) [inline CHANGELOG.md]: Shouldn't this line not be there?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,9 +2,39 @@
...
+ - [Test Groups that include Heavyweight Tests that are not run by default](https://github.com/joyent/java-manta/issues/483) 
+ - [Invalid MantaUser results in 500 Internal Server Error](https://github.com/joyent/java-manta/issues/123)
+ - [Change all hard coded "/" values to MantaClient.SEPARATOR ](https://github.com/joyent/java-manta/issues/363)  
+### Changed
+ - [Upgraded upstream dependencies to latest stable versions:](https://github.com/joyent/java-manta/pull/485)
+     - HTTP Signatures dependency: 4.0.9 → 4.0.10
```
Comment (dekobon) [inline CHANGELOG.md]: Please link to the commit and not dump all of the versions.

**Comment by nairashwin952013** [comment]
This branch will only be merged after the workin [branch](https://github.com/nairashwin952013/java-manta/tree/ashwin-fix-488) tracking my current progress towards fixing [Issue#488](https://github.com/joyent/java-manta/issues/488) is squashed and merged into the main repository.

File: CHANGELOG.md (hunk 2 above)
Comment (nairashwin952013) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 1 above)
Comment (nairashwin952013) [reply CHANGELOG.md]: Didn't get you ?

Comment (dekobon) [reply CHANGELOG.md]: This isn't the same file that I commented on. Perhaps github had a consistency issue.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,9 +2,16 @@
...
-## [3.3.2] -
+## [3.4.0] -
+### Added
+ - [Updated Prune-Depth Documentation. Add Unit Tests for more limit values](https://github.com/joyent/java-manta/pull/484)
+### Changed
+ - [Upgraded dependencies to latest stable versions:](https://github.com/joyent/java-manta/commit/1ea4694b2747c42fa7b8b60733fbd7699a3f2426)
 ### Fixed
- - [Content-Type is not set to application/octet-stream when doing CSE](https://github.com/joyent/java-manta/issues/486) 
+ - [Content-Type is not set to application/octet-stream when doing CSE](https://github.com/joyent/java-manta/issues/486)
+ - [Test Groups that include Heavyweight Tests that are not run by default](https://github.com/joyent/java-manta/issues/483)
```
Comment (dekobon) [inline CHANGELOG.md]: Is this change included? I think it isn't, right?

Comment (nairashwin952013) [reply CHANGELOG.md]: Removed.

**Comment by nairashwin952013** [comment]
This PR has been squashed and merged into the main repository.

**Pull request merged**